javascript-typing-tutor/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Typing Tutor</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="word">
+      <span class="underscore">p</span>
+      <span>l</span>
+      <span>e</span>
+      <span>a</span>
+      <span>s</span>
+      <span>e</span>
+      <span>&nbsp;</span>
+      <span>t</span>
+      <span>y</span>
+      <span>p</span>
+      <span>e</span>
+      <span>&nbsp;</span>
+      <span>t</span>
+      <span>h</span>
+      <span>i</span>
+      <span>s</span>
+      <span>&nbsp;</span>
+      <span>s</span>
+      <span>e</span>
+      <span>n</span>
+      <span>t</span>
+      <span>e</span>
+      <span>n</span>
+      <span>c</span>
+      <span>e</span>
+    </div>
+    <script>
```

Comment:
you should move all the javascript code inside the `main.js` file and link it with the `script` tag here instead :)